build/Targets/Packages.props

```diff
@@ -47,8 +49,10 @@
     <MicrosoftBuildTasksCoreVersion>15.1.0-preview-000458-02</MicrosoftBuildTasksCoreVersion>
     <MicrosoftCodeAnalysisAnalyzersVersion>$(RoslynDiagnosticsNugetPackageVersion)</MicrosoftCodeAnalysisAnalyzersVersion>
     <MicrosoftCodeAnalysisBuildTasksVersion>2.0.0-rc2-61102-09</MicrosoftCodeAnalysisBuildTasksVersion>
+    <MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitVersion>$(MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitVersion>
```

Comment:
Can we avoid language specific properties here? I see that the values for both C# and VB properties are identical in this file, and I don't see any reason why they will ever be different between languages in future.

Reply:
> Can we avoid language specific properties here?

We cannot. The property name is dictated by Build Boss and the NuGet package name we are referencing. The extracted shared property (the value of the property seen here) is the limit of our ability.